src/sentry/runner/commands/init.py

```diff
@@ -12,9 +12,10 @@
 
 
 @click.command()
+@click.option('--dev/--no-dev', default=False, help='Use settings more conducive to local development.')
```

Comment:
It's the suggested way of setting up a boolean flag. Does adding in some random ``--prod`` or whatever sound better?